src/apps/applets/domain/constants.py

```diff
@@ -24,3 +24,11 @@ class ManagersRole(str, Enum):
     MANAGER = "manager"
     COORDINATOR = "coordinator"
     EDITOR = "editor"
+
+
+class DataRetention(str, Enum):
+    INDEFINITELY = "indefinitely"
+    DAY = "day"
+    WEEK = "week"
+    MONTH = "month"
+    YEAR = "year"
```

Comment:
I think that from the english language perspective would be better next format:
```suggestion
class DataRetention(str, Enum):
    INDEFINITELY = "indefinitely"
    DAILY = "daily"
    WEEKLY = "weekly"
    MONTHLY = "monthly"
    ANNUALLY = "annually"
```

Reply:
That comes from acceptance criteria of the story: https://mindlogger.atlassian.net/wiki/spaces/MINDLOGGER1/pages/162922497/AUS-4.1.4.2+-+Applet+settings+tab+of+the+Applet+page+Data+Retention

Reply:
Alright, but also according to the AC you should use plural
![Google Chrome 2023-03-02](https://user-images.githubusercontent.com/117179957/222399092-5b7fbbdf-e15c-4490-a856-ad4bf1b11e20.png)

Reply:
Yeah, that i fixed